tests/basic.rs

```diff
@@ -753,18 +754,6 @@ async fn address_send_exercises_backpressure() {
 
     context.yield_once(&mut Greeter).await; // process one message
 
-    // Sink
-    let mut sink = address.into_sink();
-    let _ = sink
-        .send(PrintHello("world"))
-        .now_or_never()
-        .expect("be able to queue another message because the mailbox is empty again");
-
-    assert!(
-        sink.send(PrintHello("world")).now_or_never().is_none(),
-        "Fail to queue 2nd message because mailbox is full"
-    );
```

Comment:
I've deleted this test because the implementation is so simply, I don't think we need to test it and the existence of the method is tested via the example.

Reply:
Agreed